python/pyarrow/public-api.pxi

```diff
@@ -118,6 +119,8 @@ cdef api object pyarrow_wrap_data_type(
         cpy_ext_type = dynamic_cast[_CPyExtensionTypePtr](ext_type)
         if cpy_ext_type != nullptr:
             return cpy_ext_type.GetInstance()
+        elif ext_type.extension_name() == tensor_name:
```

Comment:
Just wondering, does it work as well to just do here `.. == b"arrow.fixed_shape_tensor"` (that would avoid having to define the name earlier in the code, which is a bit easier to read)

Reply:
Works, and it is so much nicer!